build.gradle

```diff
@@ -78,12 +107,54 @@ jar {
                 'Implementation-Vendor': 'Broad Institute',
                 'Implementation-Version': version
     }
+    from('src/main/resources/properties.templates') {
```

Comment:
This repeated copy block is so ugly.  I'm trying to come up with a way around it but I'm not sure how to do it in a way that isn't worse.